lib/DDG/Spice/Aur.pm

```diff
@@ -1,30 +1,41 @@
 package DDG::Spice::Aur;
+# ABSTRACT: Write an abstract here
+# Start at https://duck.co/duckduckhack/spice_overview if you are new
+# to instant answer development
 
-use strict;
```

Comment:
Please keep this in. Aside from the actual benefits of `use strict`, right now Github is having problems detecting the language we're using (Perl vs Perl6) so `use strict` ensures our code has proper Perl5 syntax highlighting. They currently don't highlight Perl6 code :disappointed: